### PR TITLE
Add QuestDB @ 2.1.3

### PR DESCRIPTION
**Pull request by BrandonEscamilla**

### CODE REVIEW

File: Q/c_questdb_client/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/c-questdb-client/questdb-rs-ffi
+cargo build --release
+if [[ "${target}" == *-w64-mingw32* ]]; then    
+    install -D -m 755 "target/${rust_target}/release/questdb_client.${dlext}" "${libdir}/c_questdb_client.${dlext}"
+else
+    install -D -m 755 "target/${rust_target}/release/libquestdb_client.${dlext}" "${libdir}/c_questdb_client.${dlext}"
+fi
```
Comment (giordano) [inline Q/c_questdb_client/build_tarballs.jl:20]: I'd simplify to
```suggestion
install -D -m 755 "target/${rust_target}/release/*questdb_client.${dlext}" "${libdir}/libquestdb_client.${dlext}"
```
and call the product below `libquestdb_client`

File: Q/c_questdb_client/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+"""
+#only macos
+platforms = supported_platforms(exclude=Sys.islinux)
+# Our Rust toolchain for i686 Windows is unusable
+filter!(p -> !Sys.iswindows(p) || arch(p) != "i686", platforms)
+
+# The products that we will ensure are always built
+products = [    
+    LibraryProduct("c_questdb_client", :c_questdb_client)
```
Comment (giordano) [inline Q/c_questdb_client/build_tarballs.jl:33]:
```suggestion
    LibraryProduct("libquestdb_client", :libquestdb_client)
```

**Review by giordano — COMMENTED** [review]
Thanks for your contribution!

**Comment by BrandonEscamilla** [comment]
Thank you for the suggestions @giordano!

File: Q/c_questdb_client/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/c-questdb-client/questdb-rs-ffi
+cargo build --release
+install -D -m 755 "target/${rust_target}/release/"*questdb_client."${dlext}" "${libdir}/libquestdb_client.${dlext}"
+install -D -m 755 "${WORKSPACE}/srcdir/c-questdb-client/include/questdb/ilp/line_sender.h" "${includedir}/line_sender.h"
+"""
+
+platforms = supported_platforms(exclude=Sys.islinux)
```
[the remark this answers is not in the document]
Comment (giordano) [reply Q/c_questdb_client/build_tarballs.jl:20]: Let's try to build also for linux and see what's the problem, if any
```suggestion
platforms = supported_platforms()
```

**Comment by BrandonEscamilla** [comment]
Thank you @giordano!

**Pull request merged**